content/en/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: blog
+title: "k8s.gcr.io Redirect to registry.k8s.io - What You Need to Know"
+date: 2023-03-10T17:00:00.000Z
+slug: image-registry-redirect
+---
+
+**Authors**: Bob Killen (Google), Davanum Srinivas (AWS), Chris Short (AWS), Frederico Muñoz (SAS
+Institute), Tim Bannister (The Scale Factory), Ricky Sadowski (AWS), Grace Nguyen (Expo), Mahamed
+Ali (Rackspace Technology), Mars Toktonaliev (independent), Laura Santamaria (Dell), Kat Cosgrove
+(Dell)
+
+
+On Monday, March 20th, the k8s.gcr.io registry [will be redirected to the community owned
+registry](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/),
+**registry.k8s.io** .
+
+
+## TL;DR: What you need to know about this change
+
+- On Monday, March 20th, traffic from the older k8s.gcr.io registry will be redirected to
+  registry.k8s.io with the eventual goal of sunsetting k8s.gcr.io.
+- If you run in a restricted environment, and apply strict domain name or IP address access policies
+  limited to k8s.gcr.io, **the image pulls will not function** after k8s.gcr.io starts redirecting
+  to the new registry. 
+- A small subset of non-standard clients do not handle HTTP redirects by image registries, and will
+  need to be pointed directly at registry.k8s.io.
+- The redirect is a stopgap to assist users in making the switch. The deprecated k8s.gcr.io registry
+  will be phased out at some point. **Please update your manifests as soon as possible to point to
+  registry.k8s.io**.
+- If you host your own image registry, you can copy images you need there as well to reduce traffic
+  to community owned registries.
+
+If you think you may be impacted, or would like to know more about this change, please keep reading.
+
+## Why did Kubernetes change to a different image registry?
```

Comment:
I would move the background and rationale sections to the end and move the sections on what is impacted, how do I know, and what do I do about it to the top.  In short, assume people will get fatigued and stop reading, make the most actionable parts first.  The history and funding and future of k8s.gcr.io is the least important part of this.